docs/_includes/footer.html

```diff
@@ -5,12 +5,23 @@
 
         {% if page.url != 'search/index' %}
         <div class="block block__sub">
-            <small>
-                <strong>Page last edited:</strong>
-                <time datetime="{{ page.last_modified_at }}">
-                    {{ page.last_modified_at | date: "%l:%M:%S %p %b %d, %Y" }}
-                </time>
-            </small>
+            <p>
+                <small>
+                    <strong>Help improve this page:</strong>
+                    <a href="https://github.com/cfpb/design-system/issues/new?assignees=&labels=content%2C+help+wanted&template=component-needs-content-edits.md&title=Suggestions+for%3A+%22{{ page.title }}%22">
+                        File an issue on GitHub
+                    </a>.
```

Comment:
This is putting a space before the period.

<img width="330" alt="Screen Shot 2020-05-07 at 1 06 48 PM" src="https://user-images.githubusercontent.com/1060248/81323674-ba4d1f80-9063-11ea-80f7-bf694729f83e.png">

Reply:
Dang it. Maybe I'll include the period in the link.